### ISSUE TITLE
Fastest way to run svaba

### ISSUE THREAD

**Issue by jsmedmar** [walaj/svaba#99]
First thank you for such a great tool.

Let's say IO throughput is >5gb/s, total ram is 500GB, and you have up to 32 cpus, what is the fastest way to run svaba?

Currently my best bet is `-p 32 --hp`, but I was wondering if that's the best approach.

Thank you so much in advance

**Comment by walaj** [comment]
Yes, that's reasonable although you may see limitations in performance with that many cores, but as you say depends on i/o rate. ~25 cores is the most I've tested on, seems to work just fine. Depending on the quality of the library, svaba can be very heavy i/o because it's looking up mate-regions if 3 or more reads point to another region. For low-quality libraries with a lot of spurious mapping, this can be expensive. 

HP flag just keeps more in memory before dumping it to disk, which prevents threads from having to pause when one of them is writing to a common file. Given your memory, this should be a great flag. 

There are various other parameters that could make it run faster, but they also potentially alter sensitivity (e.g. the -L flag which is described above).  

BTW, I just merged in a fix to another issue that can come up with massively parallel runs, so recommend that you pull and merge the latest version in ``main``.
